src/arguments.rs

```diff
@@ -40,6 +40,7 @@ pub enum Other_commands {
     login,
     token,
```

Comment:
Can you also update the `token` command and mark is at deprecated ? Pointing to the new command would also be helpful for users :)